### PR TITLE
Added import rule.

### PR DESCRIPTION
**Pull request by Monteth**
Summary | 
:-----: | :-----:
Rule name: | Import/order
Kind: | Add rule
Fixable? | Yes*
Link to docs: | https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

### Why?
This change was originally proposed by @radekmie on Slack at #paczki-i-narzędzia channel.
All participants were in favor of setting _some_ or _any_  sorting rule and there was no significant opposition or decisive support for one rule, so I decided to implement and put to an official vote original @radekmie proposal.

Rule enforces division imports into alphabetically sorted (case sensitive) groups with one (or more) newline after each. 

Warning 1: This rule does not include sort in named imports.
Warning 2: * Docs says that `--fix` option should work for all cases, but my investigation led me to the conclusion that it can not work for _some_ existing newlines in imports. 

### Examples of BAD code for this rule:

```javascript

import { omit } from 'lodash/fp';
import funaaa from './helper3';
import funAaa from './helper4';

import React from 'react';

import { funB, funa } from './helpers';
import { funb, funA } from './helpers2';
import Component from './Component';

```

### Examples of GOOD code for this rule:

```javascript

import { omit } from 'lodash/fp';
import React from 'react';

import Component from './Component';
import funaaa from './helper3';
import funAaa from './helper4';
import { funa, funB } from './helpers';
import { funA, funb } from './helpers2';

```

---
Requesting feedback from: @vazco/developers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "eslint": "6.6.0",
     "eslint-config-prettier": "6.4.0",
     "eslint-plugin-babel": "5.3.0",
+    "eslint-plugin-import": "^2.20.2",
```
Comment (MacRusher) [inline package.json]: Please use a strict version for dependency, and add a loose one to `peerDependencies`

File: configs/default.yaml [hunk 2]
```diff
@@ -30,6 +31,11 @@ rules:
   eqeqeq: 2
   func-names: 1
   guard-for-in: 2
+  import/order:
+    - error
+    - groups: [[builtin, external, internal], [parent, sibling, index], [unknown]]
+      newlines-between: always
+      alphabetize: {order: asc}
```
Comment (MacRusher) [inline configs/default.yaml]: While this is correct, IMHO using yaml all the way down produces more readable config:

```
  import/order:
    - error
    - groups:
      - - builtin
        - external
        - internal
      - - parent
        - sibling
        - index
      - - unknown
      newlines-between: always
      alphabetize:
        order: asc
```

Comment (Monteth) [reply configs/default.yaml]: When I was writing this PR i wanted to stick with the current style - use brackets and stay in one line as much as possible. At the same time I personally would opt for version you proposed. 
As long as this change doesn't affect essence of this PR and it's only semantic issue, we can make an appointment that if no one objects your proposal to the end of the vote I will implement it here.

**Review by radekmie — APPROVED** [review]

**Comment by Monteth** [comment]
Semantic changes requested by @MacRusher have been implemented.
Thank you @MacRusher for help.

**Review by MacRusher — APPROVED** [review]

**Pull request merged**